Added a animated design using pygame

IshankAgarwal opened the pull request:
Creating random intresting design with pygame.

IshankAgarwal committed:
Create README.md

IshankAgarwal committed:
Add files via upload

Animation,design with pygame

IshankAgarwal committed:
Update README.md

IshankAgarwal committed:
Add files via upload

Design output

IshankAgarwal committed:
Update README.md

IshankAgarwal committed:
Delete output_pygame design.PNG

IshankAgarwal committed:
Add files via upload

Design Output

IshankAgarwal committed:
Update README.md

themonkeyhacker commented:
Sorry dude this is cheating, marked as invalid. Be warned that Hacktoberfest may be working on a system to autoban those with enough invalid PRs. You have this exact code on a 12 month old repo you did not learn anything from this

IshankAgarwal commented:
Actually I have changed my old code. I have removed all redundant things. And added a new design also to it.
Will this is also incorrect. Guide me through this.

IshankAgarwal commented:
> Sorry dude this is cheating, marked as invalid. Be warned that Hacktoberfest may be working on a system to autoban those with enough invalid PRs. You have this exact code on a 12 month old repo you did not learn anything from this

I have changed the code, this code is different from previous one, modified it, removed redundant lines and added a new design. You may please recheck the output to both the codes. No problem,if not accepted then also. Just want to know if this modified one will work or not. Thanks
